uint8 TASK_NOT_SET = 0
uint8 NAVIGATION_CHANNEL = 1
uint8 MAG_ROUTE = 2
uint8 DOCKING = 3
uint8 SPEED_RUN = 4
uint8 RETURN_HOME = 5

uint8 current_task